Products/ZenEvents/SyslogProcessing.py

```diff
@@ -114,13 +41,30 @@ def __init__(self,sendEvent,minpriority,parsehost,monitor,defaultPriority):
         @type monitor: string
         @param defaultPriority: priority to use if it can't be understood from the received packet
         @type defaultPriority: integer
+        @param syslogParsers: configureable syslog parsers
+        @type defaultPriority: list
         """
         self.minpriority = minpriority
         self.parsehost = parsehost
         self.sendEvent = sendEvent
         self.monitor = monitor
         self.defaultPriority = defaultPriority
-
+        self.compiledParsers = self.compiledParsers(syslogParsers)
+
+    def compiledParsers(self, parsers):
+        compiledParsers = []
+        for regex in parsers.split('\n'):
```

Comment:
lets move to json so we can get tuples in the list of parsers

Reply:
Config format updated to a JSON format.